src/concrete/ml/onnx/onnx_impl_utils.py

```diff
@@ -223,3 +224,26 @@ def onnx_avgpool_compute_norm_const(
         norm_const = numpy.prod(kernel_shape)
 
     return norm_const
+
+
+def rounded_comparison(
+    x: numpy.ndarray, y: numpy.ndarray, lsbs_to_remove: int, operation: Callable
+) -> Tuple[bool]:
+    """Comparison operation using AutoRounder.
```

Comment:
we don't use `AutoRounder` here right ? `lsbs_to_remove` is an int

Reply:
yes, the idea is to avoid Autorounder since it can't work well in our case